File: esercizi_java/test/compito_python_4/Teoria.py
from Corso import Corso


class Teoria(Corso):
    def __init__(self, cod, nomecorso, nomedoc, codaula, oreset, orelab):
        super().__init__(cod, nomecorso, nomedoc)
        self._codaula = codaula
        self._oreset = oreset
        self._orelab = orelab

    def toString(self):
        return super().toString() + "\t teoria \t" + str(self._codaula) + "\t" + str(self._oreset) + "\t" + str(self._orelab)

File: esercizi_java/NetBeansProjects/Universita/src/universita/Teoria.java
package universita;

/**
 *
 * @author tobiasacchetto
 */
public class Teoria extends Corso{
    
    private int codaula, oreset;
    private double orelab;
    
    public Teoria(int cod,String nomecorso,String nomedoc,int codaula,int oreset,double orelab){
        super(cod,nomecorso,nomedoc);
        this.codaula=codaula;
        this.orelab = orelab;
        this.oreset = oreset;
    }
    
    @Override
    public String toString(){
        return super.toString()+"\tteoria\t"+codaula+"\t"+oreset+"\t"+orelab;
    }
}
